app/controllers/bike_share_app.rb

```diff
@@ -1,5 +1,41 @@
 class BikeShareApp < Sinatra::Base
+  get '/' do
+    erb :index
+  end
 
+  get '/stations' do
+    @stations = Station.all
+    erb :'stations/index'
+  end
 
+  get '/stations/new' do
+    erb :'stations/new'
+  end
+
+  post '/stations' do
+    @station = Station.create(params[:station])
+    redirect :"stations/#{@station.id}"
+  end
+
+  get '/stations/:id' do
+    @station = Station.find(params[:id])
+    erb :'stations/show'
+  end
+
+  get '/stations/:id/edit' do
+    @station = Station.find(params[:id])
+    erb :'stations/edit'
+  end
+
+  put '/stations/:id' do
+    @station = Station.update(params[:id])
+    redirect :"stations/#{@station.id}"
+  end
+
+  delete '/stations/:id' do
+    @station = Station.find(params[:id])
+    @station.destroy
```

Comment:
Can also do this in a single call:

```ruby
Station.destroy(params[:id])
```

Also, don't need to be saving this to an instance variable since you're not using it in a view that you're rendering.